src/ontology/doid-edit.owl

```diff
@@ -40307,6 +40318,88 @@ AnnotationAssertion(rdfs:label obo:DOID_0060911 "karyomegalic interstitial nephr
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060914 "This autosomal recessive systemic autoinflammatory disorder is characterized by early childhood onset of annular erythematous plaques on the face and extremities with subsequent development of partial lipodystrophy and laboratory evidence of immune dysregulation. More variable features include recurrent fever, severe joint contractures, muscle weakness and atrophy, hepatosplenomegaly, basal ganglia calcifications, and microcytic anemia.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060914 "OMIM:256040")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060914 "PRAAS1")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060914 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060914 "DOID:0060914")
+AnnotationAssertion(rdfs:label obo:DOID_0060914 "Proteasome-associated autoinflammatory syndrome 1"@en)
+SubClassOf(obo:DOID_0060914 obo:DOID_0050737)
+SubClassOf(obo:DOID_0060914 obo:DOID_0060919)
+SubClassOf(obo:DOID_0060914 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060916 (Proteasome-associated autoinflammatory syndrome 3)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060916 "Proteasome-associated autoinflammatory syndrome-3 is an autosomal recessive syndrome with onset in early infancy. Affected individuals present with nodular dermatitis, recurrent fever, myositis, panniculitis-induced lipodystrophy, lymphadenopathy, and dysregulation of the immune response, particularly associated with abnormal type I interferon-induced gene expression patterns. Additional features are highly variable, but may include joint contractures, hepatosplenomegaly, anemia, thrombocytopenia, recurrent infections, autoantibodies, and hypergammaglobulinemia. Some patients may have intracranial calcifications.
+
+https://pubmed.ncbi.nlm.nih.gov/?cmd=link&linkname=pubmed_pubmed&from_uid=26524591")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060916 "OMIM:617591")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060916 "PRAAS3")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060916 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060916 "DOID:0060916")
+AnnotationAssertion(rdfs:label obo:DOID_0060916 "Proteasome-associated autoinflammatory syndrome 3"@en)
+SubClassOf(obo:DOID_0060916 obo:DOID_0050737)
+SubClassOf(obo:DOID_0060916 obo:DOID_0060919)
+SubClassOf(obo:DOID_0060916 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060917 (facioscapulohumeral muscular dystrophy 3)
+
+AnnotationAssertion(dc:description obo:DOID_0060917 "Facioscapulohumeral muscular dystrophy-3 (FSHD3) is a digenic muscle disorder characterized by adult onset of proximal muscle weakness affecting the face, neck, scapular muscles, and upper and lower limbs. Muscle involvement is usually asymmetric, and other muscle groups may become involved with progression of the disease")
```

Comment:
Fix definition by reducing length and fitting to pattern: "A(n) {parent term} characterized by {symptoms, phenotypes, onset} that has_material_basis_in {genetic cause}."

e.g. This definition would be something like: "A facioscapulohumeral muscular dystrophy characterized by adult onset of proximal muscle weakness affecting the face, neck, scapular muscles, and upper and lower limbs that has_material_basis_in the combination of a homozygous mutation in the LRIF1 gene on chromosome 1p13 and presence of a haplotype on chromosome 4 that is permissive for DUX4 expression."

It could include something about progression or the asymmetric nature of the disease if the original publications indicate that these occur in a _majority of cases_.

Reply:
Also include definition sources (usually original PubMed publications).

Reply:
Changes have been implemented.